.github/workflows/main.yml

```diff
@@ -13,15 +13,21 @@ jobs:
       matrix:
         python-version: ["3.8", "3.9", "3.10", "3.11", "3.12", "3.13"]
```

Comment:
3.8 is end of life -- can we take it out?

Reply:
Yep, overlooked, thank you.